packages/utils/jest.config.js

```diff
@@ -6,4 +6,10 @@ module.exports = {
     '^.+\\.ts$': 'ts-jest',
     '^.+\\.js$': 'ts-jest',
   },
+  globals: {
+    'ts-jest': {
+      tsconfig: '<rootDir>/tsconfig.test.json',
+    },
+    __DEBUG_BUILD__: true,
+  },
```

Comment:
How come this needs to be repeated here, given that it already exists in the main jest config?

Reply:
Yup that's unnecessary. I think I used the utils tests to debug what I need to change. Removed. d68d3e74444e4be4aabdc8d3223f87211e304710